tests/test_terminal.cpp

```diff
@@ -1,27 +1,27 @@
 #define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 #include "doctest.h"
 
-#include <cpp-terminal/terminal.h>
+#include "../cpp-terminal/private/conversion.hpp"
```

Comment:
This change shouldn't be necessary? I think the `<cpp-terminal/terminal.h>` should be the preferred way.

Reply:
The problem is that the UTF-8 conversions are now in a seperate header file. The terminal.h header is removed soon.

Reply:
Why not `<cpp-terminal/private/terminal.h>`